crates/optimism/txpool/src/supervisor/client.rs

```diff
@@ -145,7 +146,10 @@ impl<'a> IntoFuture for CheckAccessListRequest<'a> {
     fn into_future(self) -> Self::IntoFuture {
         let Self { client, inbox_entries, executing_descriptor, timeout, safety } = self;
         Box::pin(async move {
-            tokio::time::timeout(
+            let metrics = OpTxPoolMetrics::default();
```

Comment:
we need this to live in a persistent scope. it needs to be stored in the `SupervisorClient` struct